src/components/ChatScreenHeader.js

```diff
@@ -16,6 +17,15 @@ ChatScreenHeader.propTypes = {
   room_name: propTypes.string.isRequired,
 };
 
-const StyledChatScreenHeader = styled.div``;
+const StyledChatScreenHeader = styled.div`
+border:2px solid #c7ccec;
+margin-Top:10px;
+margin-Bottom:10px;
+padding:5px;
+background-color:#c7ccec;
```

Comment:
Please use responsive units (rem) instead of pixels.

Also please use color values from theme file